### ISSUE TITLE
Improving documentation and having more information

### ISSUE THREAD

**Issue by narendraj9** [infinity-aps/infinity_aps#1]
It would be great if there was more information about the project here. For example, how things are architecture and how someone who has no idea about things can have some pointers to understand the code. 

Thanks for doing this! :)

**Comment by tmecklem** [comment]
@narendraj9 you're absolutely right! Most of the effort so far has been building out the github.com/infinity-aps/pummpcomm repo with tests and documentation, but this project must have clear communication, project goals, and community standards. It's not an oversight. Rather it's a limitation of having a limited amount of one person's time to dedicate to the project.

I understand the request loud and clear, and I'll push out a rough draft as soon as I finish work on what I'm currently working. I also tagged this as `help wanted` in case it's something someone is willing to take on. It'll require a handful of conversations to share what's in my brain, but it's another possibility.
